src/features/survey-forms/[id]/survey-form-table.tsx

```diff
@@ -225,70 +279,88 @@ export const SurveyFormTable = () => {
+                                          maxLimits[question.id ?? 0])
+                                    }
+                                  />
+                                </div>
+                                <h5 className='text-sm font-bold tracking-tight text-gray-900 dark:text-white'>
+                                  {choice.answer_text ?? ""}
+                                </h5>
+                              </div>
+                              <div
+                                className={`mb-1 ml-2.5 pr-2.5 text-gray-700 text-xs dark:text-gray-400 px-3 break-words`}
+                              >
+                                {choice.answer_description != null &&
+                                choice.answer_description.length > 55 ? (
+                                  <Tooltip placement='top' wFit={false} size='medium'>
+                                    <Tooltip.Trigger>
+                                      <p>{`${choice.answer_description.slice(0, 55)}...`}</p>
+                                    </Tooltip.Trigger>
+                                    <Tooltip.Content>
+                                      <p className=' break-words whitespace-pre-wrap'>
+                                        {choice.answer_description ?? ""}
+                                      </p>
+                                    </Tooltip.Content>
+                                  </Tooltip>
+                                ) : (
+                                  <p>{choice.answer_description}</p>
+                                )}
+                              </div>
+                              <div className='font-bold ml-2.5 px-3 text-xs'>
+                                Price: Php {choice.amount}
                               </div>
-                              <h5 className='text-xl font-bold tracking-tight text-gray-900 dark:text-white'>
-                                {choice.answer_text ?? ""}
-                              </h5>
                             </div>
-                            <p className='mb-1 ml-2.5 font-normal text-gray-700 dark:text-gray-400 whitespace-pre-wrap px-3 break-words'>
-                              {choice.answer_description ?? ""}
-                            </p>
-                            <div className='font-bold ml-2.5 px-3'>Price: {choice.amount}</div>
                           </div>
                         </div>
-                      </div>
-                    </label>
-                  ))}
+                      </label>
+                    ))}
+                  </div>
                 </div>
               </div>
             ))}
           </>
         )}
       </div>
       <div className='flex justify-between'>
-        <Button variant='primaryOutline' onClick={toggleCancelDialog}>
-          Cancel & Exit
+        <Button variant='primaryOutline' onClick={toggleBackDialog}>
+          Back
```

Comment:
@emmaacha here can only update to Back only if submitted na? then retain lang sa previous nga Cancel & Exit if ongoing pa or not submitted. also no modal if back button